lib/src/main/java/co/nimblehq/alpine/lib/mrz/MrzManager.kt

```diff
@@ -0,0 +1,145 @@
+@file:JvmName("MrzManager")
+
+package co.nimblehq.alpine.lib.mrz
+
+import android.graphics.BitmapFactory
+import android.util.Log
+import co.nimblehq.alpine.lib.mrz.MrzProcessorException.*
+import com.google.mlkit.common.MlKitException
+import com.google.mlkit.vision.common.InputImage
+import com.google.mlkit.vision.text.*
+import com.google.mlkit.vision.text.latin.TextRecognizerOptions
+import java.io.File
+import java.util.regex.Pattern
+
+interface ResultListener {
+    fun onSuccess(mrzInfo: MrzInfo)
+
+    fun onError(e: MrzProcessorException)
+}
+
+sealed class MrzProcessorException : Exception() {
+    object DefaultMrzProcessorException : MrzProcessorException()
+    object InvalidMrzInfoMrzProcessorException : MrzProcessorException()
+    object MrzInfoNotFoundMrzProcessorException : MrzProcessorException()
+    object FileNotFoundMrzProcessorException : MrzProcessorException()
+    object TextNotFoundMrzProcessorException : MrzProcessorException()
+    object TextNotRecognizedMrzProcessorException : MrzProcessorException()
+}
```

Comment:
I think it would be better if we separate files for `MrzProcessorException` to a new file same for interface `ResultListener`.